homeassistant/components/scene/lutron.py

```diff
@@ -0,0 +1,60 @@
+"""
+Support for Lutron scenes.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/scene.lutron/
+"""
+import logging
+
+from homeassistant.components.lutron import (
+    LutronDevice, LUTRON_DEVICES, LUTRON_CONTROLLER)
+from homeassistant.components.scene import Scene
+from homeassistant.const import (STATE_ON, STATE_OFF)
+
+_LOGGER = logging.getLogger(__name__)
+
+DEPENDENCIES = ['lutron']
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up the Lutron lights."""
+    devs = []
+    for scene_data in hass.data[LUTRON_DEVICES]['scene']:
+        (area_name, keypad_name, device, led) = scene_data
+        dev = LutronScene(area_name, keypad_name, device, led,
+                          hass.data[LUTRON_CONTROLLER])
+        devs.append(dev)
+
+    add_entities(devs, True)
+    return True
```

Comment:
Nothing is checking this return value. Please remove the statement.

Reply:
done